Docs/source/install/hpc/pitzer.rst

```diff
@@ -0,0 +1,278 @@
+
+      .. code-block:: bash
+
+         export proj="pas2024"
+
+      Exit the ``nano`` editor with ``Ctrl`` + ``O`` (save) and then ``Ctrl`` + ``X`` (exit).
+
+      .. important::
+
+         Now, and as the first step on future logins to pitzer, activate these environment settings:
+
+         .. code-block:: bash
+
+            source $HOME/pitzer_v100_warpx.profile
+
+      Finally, since pitzer does not yet provide software modules for some of our dependencies, install them once:
+
+      .. code-block:: bash
+
+         bash $HOME/src/warpx/Tools/machines/pitzer-osc/install_v100_dependencies.sh
+
+      .. dropdown:: Script Details
+         :color: light
+         :icon: info
+         :animate: fade-in-slide-down
+
+         .. literalinclude:: ../../../../Tools/machines/pitzer-osc/install_v100_dependencies.sh
+            :language: bash
+
+
+   .. tab-item:: CPU Nodes
+
+      We use system software modules, add environment hints and further dependencies via the file ``$HOME/pitzer_cpu_warpx.profile``.
+      Create it now:
+
+      .. code-block:: bash
+
+         cp $HOME/src/warpx/Tools/machines/pitzer-osc/pitzer_cpu_warpx.profile.example $HOME/pitzer_cpu_warpx.profile
+
+      .. dropdown:: Script Details
+         :color: light
+         :icon: info
+         :animate: fade-in-slide-down
+
+         .. literalinclude:: ../../../../Tools/machines/pitzer-osc/pitzer_cpu_warpx.profile.example
+            :language: bash
+
+      Edit the 2nd line of this script, which sets the ``export proj=""`` variable.
+      For example, if you are member of the project ``pas2024``, then run ``nano $HOME/pitzer_cpu_warpx.profile`` and edit line 2 to read:
+
+      .. code-block:: bash
+
+         export proj="pas2024"
+
+      Exit the ``nano`` editor with ``Ctrl`` + ``O`` (save) and then ``Ctrl`` + ``X`` (exit).
+
+      .. important::
+
+         Now, and as the first step on future logins to pitzer, activate these environment settings:
+
+         .. code-block:: bash
+
+            source $HOME/pitzer_cpu_warpx.profile
+
+      Finally, since pitzer does not yet provide software modules for some of our dependencies, install them once:
+
+      .. code-block:: bash
+
+         bash $HOME/src/warpx/Tools/machines/pitzer-osc/install_cpu_dependencies.sh
+
+      .. dropdown:: Script Details
+         :color: light
+         :icon: info
+         :animate: fade-in-slide-down
+
+         .. literalinclude:: ../../../../Tools/machines/pitzer-osc/install_cpu_dependencies.sh
+            :language: bash
+
+
+.. _building-pitzer-compilation:
+
+Compilation
+-----------
+
+Use the following :ref:`cmake commands <building-cmake>` to compile the application executable:
+
+.. tab-set::
+   .. tab-item:: V100 GPUs
+
+      .. code-block:: bash
+
+         cd $HOME/src/warpx
+         rm -rf build_v100
+         source $HOME/pitzer_v100_warpx.profile
```

Comment:
Since we have the large `Now, and as the first step on future logins to pitzer, activate these environment settings:` block above, we usually skip repeating this here.
```suggestion
```